charts/newrelic-logging/templates/daemonset.yaml

```diff
@@ -92,8 +101,20 @@ spec:
               {{- else }}
               value: "docker,cri"
               {{- end }}
+            {{- if or (not .Values.fluentBit.persistence) (eq .Values.fluentBit.persistence.mode "hostPath") }}
             - name: FB_DB
               value: {{ .Values.fluentBit.db | quote }}
+            {{- else if eq .Values.fluentBit.persistence.mode "persistentVolume" }}
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: FB_DB
+              value: "/db/$(NODE_NAME)-fb.db"
+            {{- else }}
+            - name: FB_DB
+              value: ""
+            {{- end }}
```

Comment:
This is going to be a 🌵  comment, in fact it's just a "thinking-out-loud-comment" rather something that we have to change.

With `.Values.fluentBit.db`, we allowed the user to choose the path of the DB in the _host_ (because we were using hostPath and mounting the whole `/var` of the host). And indirectly, this also became the path of the DB as seen from inside the Fluent Bit pod. But nothing forces us to use the same path _inside_ the pod as well.

What I mean is that `FB_DB` could _always_ be `/db/$(NODE_NAME)-fb.db` from now on, even if we're using `hostPath`, as long as we mount the `fb-db` mount at `/db`. The only caveat is that during this transition,  customers may see log duplication/loss as a new DB would be created in their hosts. Also, customers who may have configured another value for `fluentBit.db` would also have some effects I don't even want to consider. 😓

Reply:
Yep, I'm skipping this to prevent undesired behaviors, although could be something good to do in a new PR / release